src/tests/functional/inference_engine/transformations/concat_reduce_fusion.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <transformations/common_optimizations/concat_reduce_fusion.hpp>
+#include <transformations/common_optimizations/nop_elimination.hpp>
+#include <transformations/init_node_info.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <ngraph/pass/constant_folding.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+
+using namespace testing;
+using namespace ov;
+
+using ConcatReduceParams = std::tuple<ov::PartialShape, int64_t>;
+class ConcatReduceFixture : public ::testing::TestWithParam<ConcatReduceParams> {
+};
+
+TEST_P(ConcatReduceFixture, ConcatReduceMaxFusion) {
```

Comment:
Please use TEST_F and TransformationsTestsF fixture. You can look at another tests.

Reply:
TEST_F and TransformationsTestsF were reused, thank you!